### ISSUE TITLE
SickChill page no longer accessible (also not after clean install)

### ISSUE THREAD

**Issue by Manfred73** [SickChill/synology-sickchill#1]
### Setup
_Package Name:_ SickChill
_Package Version:_ 20210329-1

_NAS Model:_ DS2415+
_NAS Architecture:_ Intel Atom C2538
_DSM version:_ 6.2.4-25556

### Expected behavior
Sickchill page should load.

### Actual behavior
Page cannot be found.

### Steps to reproduce
_1._ Remove old SickChill package.
_2._ Reinstall latest package from community.
_3._ After installation and package started, go to ip-address:8081

#### Package log
_find logfiles in `/var/packages/{package}/var/`, for DSM7: `/var/log/packages/{package}.log`, for DSM < 6.0: protocol in Package Center.
_
```
Sun Apr 11 18:32:46 CEST 2021
Starting sickchill command /volume1/@appstore/sickchill/env/bin/python3 /volume1/@appstore/sickchill/share/SickChill/SickChill.py --daemon --nolaunch --pidfile /volume1/@appstore/sickchill/var/sickchill.pid --config /volume1/@appstore/sickchill/var/data/config.ini --datadir /volume1/@appstore/sickchill/var/data 
/volume1/@appstore/sickchill/share/SickChill/sickchill/locale
/volume1/@appstore/sickchill/share/SickChill/sickchill/locale
/volume1/@appstore/sickchill/share/SickChill/sickchill/locale
```
For some reason SickChill stopped working on my Synology today.
I removed the synology package and reinstalled the latest synology package from the syno community.
After the installation, I restored my files from a backup.
However, when I try to access the webinterface, it says SickChill page cannot be found. In the package overview of Synology, SickChill status says it's running. There are no errors in the Sickchill.log.
Also for a fresh install (without restoring files), the page cannot be found.

**Comment by Manfred73** [comment]
*** UPDATE ****
I noticed in my backup config.ini is in /volume1/@appstore/sickchill/var but there is now a config.ini in /volume1/@appstore/sickchill/var/data.
Could it be that some data is now in the wrong place?

**Comment by Manfred73** [comment]
Problem with SickChill page no longer available solved. I read [here](https://github.com/SynoCommunity/spksrc/pull/4161) that the port was changed to 8081 (used to be 8899). After updating, the new port was not working, since it was blocked by Synology Firewall.

I restored my configuration and set port in the restored config.ini to 8081 and opend it in the Firewall. Moved all files (cache, *db* and config.ini to the data folder. All seems to be working again.
